.github/workflows/cleanup-model.yaml

```diff
@@ -0,0 +1,22 @@
+# Workflow that runs whenever a PR is closed or merged and deletes any
```

Comment:
See [here](https://github.com/ccao-data/model-res-avm/actions/runs/6791969520/job/18464454775) for an example of a successful test run of this workflow.